nano/node/active_transactions.cpp

```diff
@@ -183,6 +183,9 @@ void nano::active_transactions::block_cemented_callback (std::shared_ptr<nano::b
 				}
 			}
 		}
+		auto const & account (!block_a->account ().is_zero () ? block_a->account () : block_a->sideband ().account);
+		activate (account);
+		activate (block_a->link ());
```

Comment:
Is this function supporting legacy blocks as well? If yes, need destination as well probably. If no, then don't need `block_a->sideband ().account` check

Reply:
Added via `ledger.destination` in https://github.com/nanocurrency/nano-node/pull/2785/commits/c581d1ad6810159210bce80730ca32ad1e0b31ef , which also makes it so that it's not done for link in state receives (we don't want that, will be confirmed by the cementing process). Good catch.